data-types/objects/src/exercises/__tests__/1_assigning_values.test.js

```diff
@@ -0,0 +1,113 @@
+/*eslint-disable no-unused-expressions */
+`
+⬇️
+
+Welcome to Assigning Values
+
+This session is designed to introduce you to how assigning values to variables
+works in javascript. In javascript the typeof a value determines whether the
```

Comment:
Nitpick: `javascript` => `JavaScript` please!

Reply:
Also, on this line, are you referring to the `typeof` operator, or do you mean `type of` here?

Reply:
Couldn't make up my mind. I'll change this to "type of"